scripts/vagrantCloudBasedContainers.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -ex
```

Comment:
Looks great, I like the combination, small update to the README, detailed comment in the script header, link to the ticket for even more detail.